graphql2/graphqlapp/query.go

```diff
@@ -14,6 +14,20 @@ type Query App
 
 func (a *App) Query() graphql2.QueryResolver { return (*Query)(a) }
 
+func (a *Query) AuthLinkStatus(ctx context.Context, id string) (*graphql2.AuthLinkStatus, error) {
+	stat, err := a.AuthLinkStore.StatusByID(ctx, id)
+	if err != nil {
+		return nil, err
+	}
+	return &graphql2.AuthLinkStatus{
+		ID:        stat.ID,
+		ExpiresAt: stat.ExpiresAt,
+		Claimed:   !stat.ClaimedAt.IsZero(),
+		Verified:  !stat.VerifiedAt.IsZero(),
+		Authed:    !stat.AuthedAt.IsZero(),
+	}, nil
```

Comment:
Would it be worth mapping this in `gqlgen.yml` like we did for the notices work so we can return flat without recreating to match the schema?

Reply:
Not unless we have a reason to want the timestamps and checking the zero-values in the UI -- this is mostly to get the `boolean` values mapped from `time.Time`